WalletWasabi/WabiSabi/Crypto/Serialization/CredentialPresentationJsonConverter.cs

```diff
@@ -16,6 +15,12 @@ public class CredentialPresentationJsonConverter : JsonConverter<CredentialPrese
 		var cx1 = reader.ReadProperty<GroupElement>(serializer, "Cx1");
 		var cV = reader.ReadProperty<GroupElement>(serializer, "CV");
 		var s = reader.ReadProperty<GroupElement>(serializer, "S");
+
+		if (ca is null || cx0 is null || cx1 is null || cV is null || s is null)
```

Comment:
I don't see why `ReadProperty` should return nullable values. It would be better to make sure it doesn't return null by adding a `if(value == null) throw new Exception("blah")` in the `ReadProperty` helper.

Reply:
I'm pretty sure nulls are never welcome in this scenario so, i wouldn't create a `ReadNullableProperty`.